Mathlib/Data/Nat/Pairing.lean

```diff
@@ -23,7 +23,7 @@ It has the advantage of being monotone in both directions and sending `⟦0, n^2
 `⟦0, n - 1⟧²`.
 -/
 
-assert_not_exists Group MonoidWithZero
+assert_not_exists Monoid MonoidWithZero
```

Comment:
```suggestion
assert_not_exists Monoid
```

Reply:
Yeah, I was wondering if we wanted to dedupe that. I suppose there's no way we define `MonoidWithZero` without `Monoid`. Right? :)